### PR TITLE
Tutorial.fix with new cylc graph

### LINKED ISSUE #218: tutorial: re-write for cylc8
The tutorial will need upgrading and partially re-writing for Cylc8.

### PR DESCRIPTION
**Pull request by wxtim**
Partially addresses #218 
My comb through tutorial/scheduling.

### CODE REVIEW

File: src/tutorial/introduction.rst [hunk 1]
```diff
@@ -3,13 +3,6 @@
 Introduction
 ============
 
-.. TODO
-
-.. warning::
-
-   This section has not yet been upgraded to match Cylc 8 changes.
-
-
```
Comment (MetRonnie) [inline src/tutorial/introduction.rst:12]: Should this be deleted yet? There's still a screenshot of the Cylc <del>7</del> 6 (!) GUI below

File: src/tutorial/scheduling/integer-cycling.rst [hunk 2]
```diff
@@ -68,7 +68,7 @@ To make a workflow repeat we must tell Cylc three things:
      :term:`cycle points <cycle point>` to be numbered.
    * The ``initial cycle point = 1`` setting tells Cylc to start counting
      from 1.
-   * ``P1`` is the :term:`recurrence`. The :term:`graph` within the ``[[[P1]]]``
+   * ``P1`` is the :term:`recurrence`. The :term:`graph` within the ``P1``
      section will be repeated at each :term:`cycle point`.
```
Comment (MetRonnie) [inline src/tutorial/scheduling/integer-cycling.rst:72]: It's no longer a section
```suggestion
   * ``P1`` is the :term:`recurrence`. The ``P1`` :term:`graph`
     will be repeated at each :term:`cycle point`.
```

File: src/tutorial/scheduling/integer-cycling.rst [hunk 3]
```diff
@@ -344,7 +344,7 @@ Recurrence Sections
 .. ifnotslides::
 
    In the previous examples we made the workflow repeat by placing the graph
-   within the ``[[[P1]]]`` section. Here ``P1`` is a :term:`recurrence` meaning
+   within the ``P1`` section. Here ``P1`` is a :term:`recurrence` meaning
```
Comment (MetRonnie) [inline src/tutorial/scheduling/integer-cycling.rst:347]: As above
```suggestion
   in the ``P1`` setting. Here ``P1`` is a :term:`recurrence` meaning
```

File: src/tutorial/scheduling/integer-cycling.rst [hunk 4]
```diff
@@ -369,7 +369,7 @@ Recurrence Sections
 
 .. ifnotslides::
 
-  By default recurrences start at the
+   By default recurrences start at the
```
Comment (MetRonnie) [inline src/tutorial/scheduling/integer-cycling.rst:372]:
```suggestion
   By default, recurrences start at the
```

File: src/tutorial/introduction.rst (hunk 1 above)
Comment (wxtim) [reply src/tutorial/introduction.rst:12]: I think that _this_ section should be pretty OK. @oliver-sanders  has implemented functionality which will produce something similar (enough, IMO) to what is shown.

File: src/tutorial/scheduling/integer-cycling.rst (hunk 2 above)
Comment (wxtim) [reply src/tutorial/scheduling/integer-cycling.rst:72]: I considered this and didn't act. LGTM.

File: src/tutorial/introduction.rst (hunk 1 above)
Comment (wxtim) [reply src/tutorial/introduction.rst:12]:
```suggestion
.. TODO

.. warning::

   This section has not yet been upgraded to match Cylc 8 changes.

```

**Review by MetRonnie — APPROVED** [review]

**Review by kinow — APPROVED** [review]

**Pull request merged**